src/ocean/THCM.C

```diff
@@ -828,6 +830,12 @@ THCM::THCM(Teuchos::ParameterList& params, Teuchos::RCP<Epetra_Comm> comm) :
         col_scaling->PutScalar(1.0);
     }
 
+    for (auto& pair : paramList.sublist("Starting Parameters")) {
```

Comment:
Why is this suddenly needed? If it is not yet needed, please move it to another commit. Also, why not call `setParameters`?

Reply:
Sorry, I guess this is to make the parameters used. It changes the behaviour, however, so I'd still move it to a separate commit.

Reply:
Moved to a different commit